lib/chef-vault/item.rb

```diff
@@ -69,44 +69,47 @@ def initialize(vault, name, opts = {})
       @client_key_path = opts[:client_key_path]
     end
 
+    # private
```

Comment:
I started moving things into private but decided to keep more serious refactors for a second PR as to keep this as simple as possible. I more or less kept most interfaces into `item.rb` intact, but there is a ton of cleanup in this file I'd like to do in a followup PR.